pyclam/manifold.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, manifold: 'Manifold', argpoints: Vector, name: str, **kwargs)
         if not argpoints and self.children:
             self.argpoints = [p for child in self.children for p in child.argpoints]
         elif not argpoints:
-            raise ValueError(f"Cluster {name} needs argpoints.")
+            raise ValueError(f'Cluster {name} needs argpoints: {argpoints}')
```

Comment:
We already established there are no argpoints, logging them here seems weird

Reply:
addressed